open_event/static/admin/css/open-event.css

```diff
@@ -15,6 +15,10 @@
     margin-left: 20%;
     margin-right: 20%
 }
+.table-btn {
+    margin-top: 0;
+    margin-bottom: 0;
+}
 
 ul.form-control {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Element (ul.form-control) is overqualified, just use .form-control without element name.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2406576949/issues/source?bid=3233864&fileBranchId=3233884#l23)